examples/advanced/llm_hf/README.md

```diff
@@ -8,15 +8,15 @@ We used the [Llama-3.2-1B model](https://huggingface.co/meta-llama/Llama-3.2-1B)
 
 For PEFT, we used LoRA method, other PEFT methods (e.g. p-tuning, prompt-tuning) can be easily adapted as well by modifying the configs following [PEFT](https://github.com/huggingface/peft) examples.
 
-We would like to showcase two key points in this example:
+We would like to showcase three key points in this example:
 - Adapt local HuggingFace training scripts, both SFT and PEFT, to federated application
 - Handling large model weights (~6 GB for Llama-3.2-1B model with float32 precision for communication), which is beyond protobuf's 2 GB hard limit. It is supported by NVFlare infrastructure via streaming, and does not need any code change.
+- Use NVFlare's filter functionality to enable model quantization and precision conversion for communication, which can significantly reduce the message size and is thus important for communicating LLM updates.  
```

Comment:
suggest some change:

"which can significantly reduce the message size, making it crucial for transmitting LLM updates."

Reply:
updated